includes/gpr-class.php

```diff
@@ -226,10 +221,12 @@ public function register_block() {
     /**
      * Enqueue the block's admin assets.
      * @return void
-     * @since 3.0.0
+     * @since 2.0.0
      */
     public function enqueue_block_admin_assets() {
-        wp_enqueue_script( 'reviews-block-google-autocomplete' );
+        wp_localize_script( 'google-places-reviews-reviews-editor-script', 'googleBlockSettings', [
+            'apiKey' => $this->api_key,
```

Comment:
@JasonTheAdams just want you to verify this is secure and will never be accessible on the frontend.